### PR TITLE
Add postgres env vars to django settings

### PR DESCRIPTION
**Pull request by dchiller**
Provides fixes to #430

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# Don't Touch!!
```
[the remark this answers is not in the document]
Comment (eliwaksbaum) [reply .env:1]: well actually we don't even need this anymore with the new way of getting the static pages into nginx. But the project_name variable affects the name of the images that get built, and before we were referencing the app image in the nginx dockerfile, so this ensured that image always had the same name

**Review by eliwaksbaum — APPROVED** [review]

File: .env (hunk 1 above)
[the remark this answers is not in the document]
Comment (dchiller) [reply .env:1]: Cool -- seems like it makes sense to keep it...I think I'll just add something to that effect in the comment

[the remark this answers is not in the document]
Comment (eliwaksbaum) [reply .env:1]: well again, it's going away soon. the new nginx build no longer relies on the app image

[the remark this answers is not in the document]
Comment (napulen) [reply .env]: Cool, thank you both. When it is safe to take this one out, we can do that. It doesn't harm to leave it on for this PR, so I just edited the comment.

**Pull request merged**